Sources/OktaDirectAuth/Internal/Requests/OOBAuthenticateRequest.swift

```diff
@@ -42,6 +42,7 @@ struct OOBResponse: Codable, Equatable, HasTokenParameters {
         && abs((lhs.interval ?? 0) - (rhs.interval ?? 0)) < tolerance
         && lhs.channel == rhs.channel
         && lhs.bindingMethod == rhs.bindingMethod
+        && lhs.bindingCode == rhs.bindingCode
```

Comment:
Addresses [a comment from the previous PR](https://github.com/okta/okta-mobile-swift/pull/214#discussion_r1946880144).